src/components/RecordingSummary.vue

```diff
@@ -119,30 +214,58 @@ export default {
       type: Object,
     },
   },
+  data() {
+    return {
+      showingLocation: false,
+    };
+  },
   computed: {
     hasBattery() {
-      return this.item.other && this.item.other.batteryLevel;
+      return this.item.batteryLevel;
     },
     window: {
       get() {
         return window;
       },
     },
+    queuedForProcessing(): boolean {
+      return this.item.processingState === "Analyse";
+    },
+    processing(): boolean {
+      return (
+        this.item.processingState === "Analyse" && this.item.processingStartTime
```

Comment:
Same here, have added a processing boolean field to db which could be used here once that lands